src/main/java/org/frc5687/infiniterecharge/robot/subsystems/Spinner.java

```diff
@@ -66,23 +88,49 @@ public Spinner(OutliersContainer container) {
         }
 
         setDefaultCommand(new DriveSpinner(this));
+
+        // TODO(mike) might want to move to Constants.java ?
```

Comment:
These should probably be in Constants.java, but putting these things into a static inner class there was causing the compiler to whine at me.